01-Login/Auth0Sample/HomeViewController.m

```diff
@@ -32,18 +32,32 @@ @interface HomeViewController ()
 
 @implementation HomeViewController
 
+static BOOL logged = false;
+
 - (IBAction)showLoginController:(id)sender {
     HybridAuth *auth = [[HybridAuth alloc] init];
 
-    [auth showLoginWithScope:@"openid" connection:nil callback:^(NSError * _Nullable error, A0Credentials * _Nullable credentials) {
-        dispatch_async(dispatch_get_main_queue(), ^{
-            if (error) {
-                NSLog(@"Error: %@", error);
-            } else if (credentials) {
-                [self showAlertWithMessage:[NSString stringWithFormat:@"Success, Access Token: %@", [credentials accessToken]]];
-            }
-        });
-    }];
+    if (!logged) {
```

Comment:
Is there a Logout button? This appears to repurpose the Login button? How does the UI change, can you add some screenshots please.